packages/application-shell/src/test-utils.spec.js

```diff
@@ -0,0 +1,277 @@
+// This file tests our test-utils file #inception
```

Comment:
This file shows how consumers of  `application-kit` could use `render` from `test-utils` to test different aspects of a feature.